tools/breaking-changes/data/breaking-changes.json

```diff
@@ -22716,7 +22716,7 @@
     ],
     "isDeleted": true,
     "deletedComment": "Class CheckoutService has been removed and is no longer part of the public API.",
-    "migrationComment": "",
+    "migrationComment": "Please check migration instrictions for each of the methods you use from this class.",
```

Comment:
instructions*

but which migration instructions we are referring to?

Reply:
Thanks, I clarified it. I think we can't cover all the methods in this one comment, so I "delegated" to the injected code comments above each of the used methods.